csrankings-r.csv

```diff
@@ -1342,6 +1342,7 @@ Routo Terada,USP,http://www.ime.usp.br/~rt,NH6zsPUAAAAJ
 Roxana Geambasu,Columbia University,https://roxanageambasu.github.io,NOSCHOLARPAGE
 Roxana Girju,Univ. of Illinois at Urbana-Champaign,http://www.linguistics.illinois.edu/people/girju,NOSCHOLARPAGE
 Roy A. Ruddle,University of Leeds,https://eps.leeds.ac.uk/computing/staff/257/professor-roy-ruddle,QE6gxWYAAAAJ
+Roy Adams,Johns Hopkins University,https://royadams.github.io/,M5yRbdMAAAAJ

```

Comment:
<img width="1249" alt="Screen Shot 2024-11-04 at 10 44 08 AM" src="https://github.com/user-attachments/assets/d59185d9-c68f-4dc3-8db5-afd4b93ccf0f">

https://www.cs.jhu.edu/people/cs-affiliates/joint-and-secondary-faculty/